src/main/resources/site/processors/add-metadata.js

```diff
@@ -46,14 +53,15 @@ exports.responseProcessor = function (req, res) {
         type: isFrontpage ? "website" : "article",
         url,
         canonicalUrl,
-        image,
+        imageUrl,
         imageWidth: 1200, // Twice of 600x315, for retina
         imageHeight: 630,
         blockRobots:
             siteConfig.blockRobots || libs.common.getBlockRobots(content),
         siteVerification,
         canonical: siteConfig.canonical,
-        twitterUserName: siteConfig.twitterUsername,
+        twitterUserName: appOrSiteConfig.twitterUsername,
```

Comment:
The old code only used siteConfig. 
So it was not possible to set this in the config file at all? Why did no one find this bug?
Our readme does say it was supported? 🤷